packages/desmos-std/src/profiles/msg_routes.rs

```diff
@@ -0,0 +1,57 @@
+use cosmwasm_std::Addr;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum ProfilesMsgs {
```

Comment:
I think it lost `AppLink` and `ChainLink`.

Reply:
I think there's no point to add those messages right know since it will not be that straightforward to implement them at the moment. Since the contract is the one who signs the messages, users cannot create links easily. I suggest we keep the #[non_exhaustive] notation till we implement them in the future

Reply:
I think it is useful to create link for the contract which is the owner of subspace. I believe that it is impossible to create the link for user in the contract since the signer is contract as you said. I am okay to keep #[non_exhaustive] notation.

Reply:
@dadamu I didn't understand what do you mean by:

> the contract which is the owner of subspace. 

Do you mean that it would be good for contracts to be able to create links in the case they are set as subspace's owners/admins?

Reply:
@bragaz Yes, that is. Contract can be the subspace owner as well. Then everything about subspace can be controlled by the contract.